src/utils/rpc/rpc.functions.ts

```diff
@@ -42,26 +47,28 @@ export const RPCServerRequest =
     managerFunction: (...args: any) => Promise<any>,
     schemaValidation?: ObjectSchema<any>
   ): any =>
-    async (payload: IRPCPayload) =>
-      runWithContext(async () => {
-        let result;
-        try {
-          if (schemaValidation) {
-            await schemaValidation.validateAsync(
-              payload.params,
-              defaultValidationOptions
-            );
-          }
-
-          setContext(Global.USER, payload.user);
-          putSkipPlugins(payload.skipPlugins);
-
-          result = await managerFunction(
-            ...(payload.params ? Object.values(payload.params) : [])
+  async (payload: IRPCPayload) =>
+    runWithContext(async () => {
+      let result;
+      try {
+        if (schemaValidation) {
```

Comment:
dont use if

Reply:
this is shared code, it wasn't re-written